tests/system/runtimes/test_nuclio.py

```diff
@@ -15,8 +15,8 @@ class TestNuclioRuntime(tests.system.base.TestMLRunSystem):
 
     @staticmethod
     def _skip_set_environment():
-        # Skip to make sure project ensured in Nuclio function deployment flow
-        return True
+        # Project ensured does not create the project if it does not exist, therefore we need to build it beforehand
+        return False
```

Comment:
`_skip_set_environment` base implementation is returning False, no reason to override it if it's doing exactly the same, simply remove the implementation here and the `test_deploy_function_without_project` that is not needed anymore